draft-ietf-quic-multipath.md

```diff
@@ -1422,7 +1422,8 @@ in {{QUIC-TRANSPORT}}.
 
 This document does not specify per-path idle timeouts. An endpoint
 can decide to close a path at any time, whether the path is in active
-use or not, by sending a PATH_ABANDON frame. It is not required
+use or not, by sending a PATH_ABANDON frame. For the initial PATH_ABANDON
+frame to close a path, it is not required
 to send a PATH_ABANDON frame at any specific point in time.
 For example, an endpoint may wait until it will anyway send another frame.
```

Comment:
```suggestion
use or not. For example, an endpoint may wait to send
the initial PATH_ABANDON frame until it anyway sends another frame.
Note that the receiver of an initial PATH_ABANDON frame is, however,
required to immediately reply (see {{path-close}}).
```

Reply:
Does that work for you @huitema and @qdeconinck ?

Reply:
Works for me.